src/Components/MovieCard/MovieCard.css

```diff
@@ -1,14 +1,28 @@
 .movie-card {
-  border: 2pt solid white;
-  width: 18vw;
+  border: .5pt solid white;
+  width: 14rem;
+  height: auto;
   display: flex;
   flex-direction: column;
-  align-items: center
+  align-items: center;
+  margin: .5rem;
+}
+.movie-card:hover {
+  transition: all .75s;
+  transform: scale(1.2);
+  box-shadow: 0 0 0 99999px rgba(0, 0, 0, .5);
+}
+
+a {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
```

Comment:
The hover state is super slick!
And love that you found the solution to that card sizing/alignment 👍